Yoast/wordpress-seo
# Remove unneccessary css

## Summary


This PR can be summarized in the following changelog entry:

* Removes CSS used at the Admin pages and moves CSS files to the corresponding folder. Some CSS files that's loaded at multiple pages, is moved to a `to-be-removed` directory. If this doesn't cause any changes, this folder can be safely deleted. 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this PR
* Make sure that everything at the Dashboard, the Admin bar, Post overview page and Post edit page still look the same
* The CSS at the admin pages should be removed, ready for the redesign

## UI changes

## Documentation

## Quality assurance


Fixes #14303
